docs/advanced/Working_with_git.rst

```diff
@@ -0,0 +1,151 @@
+==================
+Working with git
+==================
+
++----------------------------------+-----------------------------------------------------+
+| Git Commands                     |                What it does                         |
++==================================+=====================================================+
+| git **clone**  <repo>            | Used to clone the repo ``git clone <repo> | <name>``|
++----------------------------------+-----------------------------------------------------+
+| git **commit**                   |  **Commit** an applied change on the given branch   |
++----------------------------------+-----------------------------------------------------+
+| git **remote**                   | `track <http://bit.ly/_fo_rk>`_ remote branch       |
```

Comment:
While I think @willingc should probably take care of the bulk of this review (as she tends to be the docs expert), I do not think we should be using bit.ly in our documentation, as we don't have control over where they point.

This seems to be part of an effort to make this table look pretty in RST plaintext. Is it possible to widen it and use original links, or something?